tests/foreman/destructive/test_rename.py

```diff
@@ -188,7 +188,7 @@ def test_negative_rename_sat_wrong_passwd(module_target_sat):
     :CaseAutomation: Automated
     """
     username = settings.server.admin_username
-    original_name = module_target_sat.hostname
+    original_name = module_target_sat.execute('hostname').stdout.strip()
```

Comment:
` module_target_sat.hostname` doesn't work here? any reason to run hostname cmd explicitly?

Reply:
yeah, as these test operate on the same "destructive" sat instance, the negative tests can get to the satellite already renamed by the positive tests, hence this change